plugins/woocommerce/includes/rest-api/Controllers/Version3/class-wc-rest-product-custom-fields-controller.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * REST API CustomFields controller
+ *
+ * Handles requests to the /products endpoint.
```

Comment:
Should this say `Handles requests to /products/custom-fields endpoint.`?

Reply:
done -> [f8e8106](https://github.com/woocommerce/woocommerce/pull/48504/commits/f8e81060177d342b769076fdf57f7b83cca2f906)